packages/component-library/src/Hero/CollectionHero.js

```diff
@@ -6,16 +6,27 @@ import { css } from 'emotion';
 const teamTitleStyle = css`
   display: block;
   font-size: 13px;
-  font-family: 'Rubik';
+  font-family: 'Rubik', sans-serif;
   text-transform: uppercase;
   letter-spacing: 3px;
 `;
 
+const titleStyle = css`
+  font-size: 50px;
+  line-height: 1.2;
+  font-weight: 300;
+  font-family: 'Rubik', sans-serif;
+  margin-bottom: 12px;
+  @media (max-width: 640px) {
+    font-size: 36px;
+  }
+`;
+
 const CollectionHero = ({ heroTitle, heroSubtitle, mainProjectColor, teamTitle }) => (
   <Hero mainProjectColor={mainProjectColor}>
     <div>
       <span className={teamTitleStyle}>{teamTitle}</span>
-      <h1 className={'Title'}>{heroTitle}</h1>
+      <h1 className={titleStyle}>{heroTitle}</h1>
```

Comment:
This was the only place in 2018 code where we were using 'Title' from the global css file - so changed it here rather than changing it globally